### ISSUE TITLE
[Paremeter Question] Meaning of magic numbers in transforms.Normalize()

### ISSUE THREAD

**Issue by kimsooyoung** [Shaka-Labs/ACT#9]
Hi, First of all thank you so much for sharing this great works.
I have a question in `policy.py`. Which is about mean and std value.

```python
class ACTPolicy(nn.Module):
    def __init__(self, args_override):
        super().__init__()
        model, optimizer = build_ACT_model_and_optimizer(args_override)
        self.model = model # CVAE decoder
        self.optimizer = optimizer
        self.kl_weight = args_override['kl_weight']
        print(f'KL Weight {self.kl_weight}')

    def __call__(self, qpos, image, actions=None, is_pad=None):
        env_state = None
        normalize = transforms.Normalize(
            mean=[0.485, 0.456, 0.406],
            std=[0.229, 0.224, 0.225]
        )
```

What is the meaning of (0.485, 0.456, 0.406) (0.229, 0.224, 0.225) and how did you get that?

Thank you in advance.

**Comment by kimsooyoung** [comment]
These numbers represent the mean and standard deviation values used for normalizing input images when using a model pre-trained on the [ImageNet](https://www.image-net.org/) dataset.
